### ISSUE TITLE
this._queries is not iterable

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode-pull-request-github#1940]
Have Pull Request nighyl extension installed.
I see this in VS Code insiders in the Developer Console:

```
TypeError: this._queries is not iterable
	at t.StateManager.<anonymous> (/Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1)
	at Generator.next (<anonymous>)
	at /Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1
	at new Promise (<anonymous>)
	at n (/Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1)
	at t.StateManager.setIssueData (/Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1)
	at t.StateManager.<anonymous> (/Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1)
	at Generator.next (<anonymous>)
	at /Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1
	at new Promise (<anonymous>)
	at n (/Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1)
	at t.StateManager.refresh (/Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1)
	at /Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1
	at l.fire (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:45)
	at /Users/isidor/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-2020.6.34021/media/extension.js:1
	at processTicksAndRejections (internal/process/task_queues.js:93)
```

**Comment by RMacfarlane** [comment]
Do you also see this with the latest version of the extension? (2020.6.41221) Have you modified the `githubIssues.Queries` setting?

**Comment by isidorn** [comment]
@RMacfarlane I have not modified the `githubIssues.Queries` setting. I had extension auto update setting and I was using PR extension nightly, so yes I think I was on latest.

**Comment by eyalmor-adsk** [comment]
Plus 1.
I see this issue as well:
Error:
```
var Module=typeof DracoDecoderModule!=="undefined"?DracoDecoderModule:{};var isRuntimeInitialized=false;var isModuleParsed=false;Module["onRuntimeInitialized"]=function(){isRuntimeInitialized=true;if(isModuleParsed){if(typeof Module["onModuleLoaded"]==="function"){Module["onModuleLoaded"](Module)}}};Module["onModuleParsed"]=function(){isModuleParsed=true;if(isRuntimeInitialized){if(typeof Module["onModuleLoaded"]==="function"){Module["onModuleLoaded"](Module)}}};function isVersionSupported(versionString){if(typeof versionString!=="string")return false;const version=versionString.split(".");if(version.length<2||version.length>3)return false;if(version[0]==1&&version[1]>=0&&version[1]<=3)return true;if(version[0]!=0||version[1]>10)return false;return true}Module["isVersionSupported"]=isVersionSupported;var moduleOverrides={};var key;for(key in Module){if(Module.hasOwnProperty(key)){moduleOverridError: abort(TypeError: this._queries is not iterable). Build with -s ASSERTIONS=1 for more info.

Error: abort(TypeError: this._queries is not iterable). Build with -s ASSERTIONS=1 for more info.    at process.abort Z~/.vscode/extensions/cesium.gltf-vscode-2.2.12/node_modules/draco3dgltf/draco_decoder_gltf_nodejs.js:35:7672)    at process.emit (events.js:208:15)    at processEmit (~/.vscode/extensions/node_modules/signal-exit/index.js:161:1)    at process.emit (~/.vscode/extensions/zignd.html-css-class-completion-1.19.0/node_modules/source-map-support/source-map-support.js:453:21)    at processPromiseRejections (internal/process/promises.js:201:33)    at processTicksAndRejections (internal/process/task_queues.js:86:32)
```

Version:
```
Version: 1.47.3
Commit: 91899dcef7b8110878ea59626991a18c8a6a1b3e
Date: 2020-07-23T13:08:29.692Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.5.0
```

**Comment by RMacfarlane** [comment]
This should have been fixed as part of 2a26ec7fb2b590ea40550c5677c81f241d7ec798, this._queries is now always initalized to an empty array.
